File: src/01-hello-world/hello.c
#include <stdio.h>
#include <stdlib.h>

#define LOL_PRINT(str, suffix) printf("%s %s\n", str, suffix)

int main(int argc, char *argv[])
{
    if (argc != 2)
    {
        LOL_PRINT(">", " lol try harder");
        exit(1);
    }

    LOL_PRINT(argv[1], " lmao");
    return 0;
}

File: src/01-hello-world/hello.cs
using System;

class HemloCS
{
    static void Main(string[] args)
    {
        if (args.Length != 1)
        {
            Console.Out.WriteLine("> try harder lol");
            Environment.Exit(1);
        }

        Console.Out.Write(args[0]);
        Console.Out.WriteLine(" lmao");
    }
}
